analyzers/maven/maven_test.go

```diff
@@ -0,0 +1,27 @@
+package maven_test
```

Comment:
We should add a test here that ensures we can accurately get the correct graph from nested pom files.

Reply:
@zlav  this still valid with the migration of the logic to `buildtools/maven`?